include/mbedtls/ssl.h

```diff
@@ -4525,12 +4529,23 @@ void mbedtls_ssl_conf_session_tickets(mbedtls_ssl_config *conf, int use_tickets)
  *        error code are then failing.
  *
  * \param conf  SSL configuration
- * \param new_session_tickets_enabled  Enable or disable
- *                                     (MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_ENABLED or
- *                                      MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_DISABLED)
+ * \param use_new_session_tickets Enable or disable
+ *                                (MBEDTLS_SSL_NEW_SESSION_TICKETS_ENABLED or
+ *                                 MBEDTLS_SSL_NEW_SESSION_TICKETS_DISABLED)
  */
 void mbedtls_ssl_conf_enable_new_session_tickets(mbedtls_ssl_config *conf,
-                                                 int new_session_tickets_enabled);
+                                                 int use_new_session_tickets);
+
+/**
+ * \brief          Get if usage of TLS 1.3 NewSessionTicket messages is enabled or not
+ *
+ * \param conf     SSL configuration
+ *
+ * \return         MBEDTLS_SSL_NEW_SESSION_TICKETS_ENABLED or
+ *                 MBEDTLS_SSL_NEW_SESSION_TICKETS_DISABLED
+ */
+int mbedtls_ssl_conf_is_new_session_tickets_enabled(const mbedtls_ssl_config *conf);
```

Comment:
Does this need to be a public function? Also, naming is inconsistent with `mbedtls_ssl_conf_get_session_tickets()`.

Reply:
ditto and yes I know the name is inconsistent but the setting function is named `mbedtls_ssl_conf_enable_new_session_tickets()` and not `mbedtls_ssl_conf_new_session_tickets()`. It is not mbedtls_ssl_conf_new_session_tickets()` because it already exists (setting the number of tickets to send after the handshake on server side). That's why I felt uncomfortable to name it `mbedtls_ssl_conf_get_new_session_tickets()` as I felt it would be associated with `mbedtls_ssl_conf_new_session_tickets()` which is not the case.

Reply:
Ah right, makes sense. Anyway, I'll care much less about the name as soon as it's a private function :)